### PR TITLE
is19 and other touchups

### PR DESCRIPTION
**Pull request by chase-stubbs**
### FEATURES
- highlighting of active selected ship during placement phase
- begin game with a ship chosen
- game over modal cannot be exited out of so that the code is more robust

### STYLING
- element borders
- font
- colors
- spacing

### CODE REVIEW

File: src/main/java/assets/css/custom.css [hunk 1]
```diff
@@ -50,164 +61,278 @@
...
+}
+
+/* I added this for coloring of the legend. LK, you will have to figure out
+   how to get the target to only be during attacking... TODO */
+.target {
+    background-color: darkred;
 }
 
+/* Is there something potentially going wrong with this? I'm not aware as to
+   what's causing our placement bug... TODO */
```
[the remark this answers is not in the document]
Comment (Bensonre) [reply src/main/java/assets/css/custom.css]: the color is not highlighted during hover for all the spaces the ship will take

File: src/main/java/assets/game.js [hunk 2]
```diff
@@ -108,39 +111,45 @@ function sendXhr(method, url, data, handler) {
...
             if (cell === undefined) {
                 // ship is over the edge; let the back end deal with it
                 break;
             }
-            cell.classList.toggle("placed");//add class placed to cell so it will show up with color on the board!
-        }
-  }
+            cell.classList.toggle("placed"); // add class placed to cell so it will show up with color on the board!
+    }
+}
```
[the remark this answers is not in the document]
Comment (chase-stubbs) [reply src/main/java/assets/game.js]: good catch

File: src/main/java/assets/css/custom.css [hunk 3]
```diff
@@ -50,164 +61,278 @@
...
+
+
+/* this color intentionally matches ship color */
 td:hover {
-    background-color: aqua;
+    background-color: #505050;
+}
+
+/* I added this for coloring of the legend. LK, you will have to figure out
+   how to get the target to only be during attacking... TODO */
```
[the remark this answers is not in the document]
Comment (chase-stubbs) [reply src/main/java/assets/css/custom.css]: like we don't want the crosshair to be on other elements during gameplay. only when clicking opponent's board

[the remark this answers is not in the document]
Comment (xMestas) [reply src/main/java/assets/css/custom.css]: Oh that should be easy enough

File: src/main/java/assets/css/custom.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (xMestas) [reply src/main/java/assets/css/custom.css]: Why not?

File: src/main/java/assets/css/custom.css [hunk 4]
```diff
@@ -50,164 +61,278 @@
...
-/*allows for a small seperation in list elements */
+
+/* allows for a small separation in list elements */
 li{
-    display:block;
-    margin: 5px;
+    display:flex;
+    flex-direction:row;
+    padding: 5px;
+    font-family: 'Kalam';
```
[the remark this answers is not in the document]
Comment (xMestas) [reply src/main/java/assets/css/custom.css]: Kalen

File: src/main/java/assets/css/custom.css [hunk 5]
```diff
@@ -1,15 +1,21 @@
-/* Add additional stylesheets below
--------------------------------------------------- */
-/*sets up the two large grids to take up as much space as we give them*/
+
+/* =======  PRIMARY GRIDS  =============================================================== */
+
```
[the remark this answers is not in the document]
Comment (xMestas) [reply src/main/java/assets/css/custom.css]: Agreed

**Comment by xMestas** [comment]
Implements #19

**Pull request merged**